### ISSUE TITLE
add 'comment' as optional field

### ISSUE THREAD

**Issue by rlzijdeman** [LinkedPasts/linked-places#19]
Please consider adding 'comment' as an optional field. Such a field would give some space for elaboration or link to more extensive documentation.

**Comment by docuracy** [comment]
I agree, comments can sometimes be very useful and should not be disallowed or break validation, wherever they appear. There is a convention of using property names beginning with an underscore for 'private' properties, so perhaps `_comment` is the way to go?
